addons/resources/src/ResourcePanel.js

```diff
@@ -0,0 +1,161 @@
+import { document, DOMParser } from 'global';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@storybook/components';
+import styled from '@emotion/styled';
+import AceEditor from 'react-ace';
```

Comment:
Note for the future: I think we need to stick to a single editor in all these code highlighting components.
- Storysource has now a `react-syntax-highlighter` which is not an editor thought,
- Here it's an `AceEditor`
- IMO @libetl is adding another one

Reply:
We are not yet decided whether to add codemirror or monaco for the moment.
The change will probably happen in the tech/overhaul-ui branch